setup.py

```diff
@@ -21,8 +21,9 @@
 long_description = Path(this_directory, 'README.md').read_text()
 
 setup(
+    python_requires='>=3.6.9',
     name='palmcli',
-    version='2.0.1',  # Don't forget to bump the version in docs/source/conf.py too!
+    version='2.0.2',  # Don't forget to bump the version in docs/source/conf.py too!
```

Comment:
Let's revert this change, version bumping to be handled in release branches from develop -> main